python/dask_cudf/dask_cudf/tests/test_distributed.py

```diff
@@ -54,7 +54,7 @@ def test_merge():
 
 
 @pytest.mark.skipif(
-    not more_than_two_gpus(), reason="Machine does not have more than two GPUs"
+    not more_than_n_gpus(2), reason="Machine does not have more than two GPUs"
```

Comment:
```suggestion
    not more_than_n_gpus(2), reason="Machine does not at least two GPUs"
```

Reply:
Just like the message was misleading, I believe the function name is too and I would suggest renaming it to `at_least_n_gpus` or something more accurate.